src/main.c

```diff
@@ -457,16 +464,8 @@ int main(int argc, char* argv[]) {
   else
     jq_set_attr(jq, jv_string("VERSION_DIR"), jv_string_fmt("%.*s-master", (int)(strchr(JQ_VERSION, '-') - JQ_VERSION), JQ_VERSION));
 
-#if (!defined(WIN32) && defined(HAVE_ISATTY)) || defined(HAVE__ISATTY)
-
-#if defined(HAVE__ISATTY) && defined(isatty)
-#undef isatty
-#define isatty _isatty
-#endif
-
-  if (!program && isatty(STDOUT_FILENO) && !isatty(STDIN_FILENO))
+  if (!program && (isatty(STDOUT_FILENO) || !isatty(STDIN_FILENO)))
```

Comment:
We didn't guard the other use of `isatty()` with an `#ifdef`.  The code assumed there's always an `isatty()`, though it might be `_isatty()`, or it might not be a `#define`, so either way we shouldn't `#ifdef isatty`, but either leave it as is or `#define` some other macro for the `#ifdef`.  Comments?